sqlglot/dialects/materialize.py

```diff
@@ -0,0 +1,58 @@
+from sqlglot import exp
+from sqlglot.dialects.dialect import NormalizationStrategy
+from sqlglot.dialects.postgres import Postgres
+from sqlglot.tokens import TokenType
+
+from sqlglot.transforms import (
+    remove_unique_constraints,
+    ctas_with_tmp_tables_to_create_tmp_view,
+    preprocess,
+)
+
+
+class Materialize(Postgres):
+    # Define any specific normalization strategies or settings here
+    NORMALIZATION_STRATEGY = NormalizationStrategy.CASE_INSENSITIVE
+
```

Comment:
According to the [docs](https://materialize.com/docs/sql/identifiers/#case-sensitivity), this override doesn't look correct - AFAICT `"FOO"` is case-sensitive and I guess unquoted identifiers are lowercased?

Most likely we should remove this and use Postgres' strategy.

Reply:
Ah yes, good catch! Thank you!